scrapers/LetsDoeIt.yml

```diff
@@ -33,16 +33,18 @@ xPathScrapers:
                 with: $1-$2-$3
           - parseDate: 2006-01-02
       Details:
-        selector: //meta[@name="description"]/@content
+        selector: //meta[@itemprop="description"]/@content
         postProcess:
           - replace:
               - regex: "&#039;"
                 with: "'"
+              - regex: (.*\.?\!?)(?:\s-\s\w.*-.*)$ # remove Studio name at the end of a description
+                with: $1
       Tags:
         Name: $details//div[@class="col"][4]//a/text()|$details//div[@class="col"][6]//a/text()
       Performers:
-        Name: $actors//span/a//text()
+        Name: $actors//span[position()>1]/a//text() # position >1 to ignore studio link
```

Comment:
```yaml
        Name: $actors//span/a[contains(@href,"/models/")]
```
also works and is probably cleaner to use?

Reply:
thanks, it is a better one, i reaplced it